gitpod-io/website
# Fix some typos in troubleshooting

## Description
This PR attempts to fix some issues in wording, grammar and typos all over the place in the troubleshooting guide for Gitpod.

## Related Issue(s)
Addresses a code review of mine in https://github.com/gitpod-io/website/pull/1853.

## How to test
Hard to say, but just try reading it and see if it looks / sounds better to you!

## Release Notes

```release-note
NONE
```


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/1886"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/1886"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>
